build.sbt

```diff
@@ -8,7 +8,7 @@ parallelExecution in Test := false
 
 parallelExecution in IntegrationTest := false
 
-scalaVersion := "2.9.1"
```

Comment:
You need to update `.travis.yaml` to reflect this.
https://github.com/tumblr/collins/blob/master/.travis.yml#L7